test/scripts/jenkins_xpack_ci_group.sh

```diff
@@ -14,11 +14,11 @@ if [[ -z "$IS_PIPELINE_JOB" ]] ; then
   echo " -> Ensuring all functional tests are in a ciGroup"
   cd "$XPACK_DIR"
   node scripts/functional_tests --assert-none-excluded \
```

Comment:
Change this whole bit to:
`node scripts/functional_tests --assert-none-excluded --include-tag ciGroup2`
and I think you should be okay